README.md

```diff
@@ -1,3 +1,5 @@
+> **Note**: We do not support running out of the git main branch. This is where development happens. Please use released versions via the published repo or git tags
+
 > **Note**: All the helm charts in this repo are beta. We encourage you to try them out and contribute. The API may change as we move towards a production ready release.
```

Comment:
```suggestion
> [!Note]
> We do not support running out of the git main branch. This is where development happens. Please use released versions via the published repo or git tags.

> [!Note]
> All the helm charts in this repo are beta. We encourage you to try them out and contribute. The API may change as we move towards a production ready release.
```

This will give some nice new markup. the old `**Note**:` isn't supported anymore.